go.mod

```diff
@@ -13,6 +13,7 @@ require (
 	github.com/kubernetes-csi/csi-test/v3 v3.0.0
 	github.com/onsi/ginkgo v1.11.0
 	github.com/onsi/gomega v1.7.1
+	github.com/prometheus/client_golang v1.0.0
```

Comment:
I'm surprised so many packages were changed but there were no changes to go.sum?

Reply:
ah maybe I missed adding it to the commit!